src/help-widget.spec.ts

```diff
@@ -390,4 +390,59 @@ describe('BBHelpHelpWidget', () => {
     helpWidget.load(fakeConfig);
     expect(helpWidget.getWhatsNewRevision()).toEqual(0);
   });
+
+  it('should hide invoker when mobile view', (done) => {
+    (window as any).innerWidth = 400;
+    helpWidget['resizeContainer']();
+
+    expect(helpWidget['invoker'].style.display).toEqual('none');
+    done();
+  });
+
+  it('should size the closed content container when mobile view', (done) => {
```

Comment:
We should add an expect case for width? Or a check for the closed class?  Something else to indicate what happens on close.